docs/en/Time-To-Live.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: global
+title: Time to Live
+nickname: Time to Live
+group: Features
+priority: 8
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio supports a `Time to Live (TTL)` setting on each file and directory in the namespace. This
+feature can be used to effectively manage the Alluxio cache, especially in environments with strict
+guarantees on the data access patterns. For example, if analytics is only done on the last week of
+ingested data, TTL can be used to explicitly flush old data to free the cache for new
+files.
+
+## Overview
+
+Alluxio has TTL attributes associated with each inode. These attributes are journaled and persist
```

Comment:
user may not be familiar with the concept of "inode", how about "file" or "directory"

Reply:
Updated, thanks.